python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_security.schema.yml

```diff
@@ -89,7 +89,9 @@ keys:
                     - disabled
                     - sha1
                     - sha256
-                    - "null"  # TODO: AVD 5.0.0
+                    - sha384
```

Comment:
I can not see these valid-values in EOS **[4.32.0F-36401836.4320F]** cli-

```
s1-leaf2(config-ipsec-sa)#esp integrity ?
  null    null security profile
  sha1    sha1 security profile
  sha256  sha256 security profile
  <cr>    

s1-leaf2(config-ipsec-sa)#esp integrity sha1 ?
  <cr>  

s1-leaf2(config-ipsec-sa)#esp integrity sha1 
s1-leaf2(config-ipsec-sa)#esp encryption ?
  aes128        AES - 128-bit Advanced Encryption Standard
  aes128gcm128  128 bit AES-GCM with 128 bit ICV
  aes128gcm64   128 bit AES-GCM with 64 bit ICV
  aes256        AES - 256-bit Advanced Encryption Standard
  aes256gcm128  256 bit AES-GCM with 128 bit ICV
  null          Null Encryption
  <cr> 
```

Reply:
it is there for 4.32.1F
```
wan1-dc1(config-ipsec-sa)#esp integrity ?
  md5     md5 security profile
  null    null security profile
  sha1    sha1 security profile
  sha256  sha256 security profile
  sha384  sha384 security profile
  sha512  sha512 security profile
  <cr>    

wan1-dc1(config-ipsec-sa)#esp encryption ?
  3des          Three key triple DES
  aes128        AES - 128-bit Advanced Encryption Standard
  aes128gcm128  128 bit AES-GCM with 128 bit ICV
  aes128gcm64   128 bit AES-GCM with 64 bit ICV
  aes256        AES - 256-bit Advanced Encryption Standard
  aes256gcm128  256 bit AES-GCM with 128 bit ICV
  null          Null Encryption
  <cr>
```